h/features/models.py

```diff
@@ -105,3 +112,57 @@ def remove_old_flags(cls):
 
     def __repr__(self):
         return '<Feature {f.name} everyone={f.everyone}>'.format(f=self)
+
+
+class Cohort(Base, mixins.Timestamps):
```

Comment:
Just wondering whether `cohort` would be expressive enough as a table name, since we're not planning to overload the term for now? But otherwise I'd slightly prefer going for `featurecohort` to keep the model and the table names lined up.